proms-resources/feature/src/main/features/feature.json

```
@@ -62,15 +62,18 @@
         // Importing visits
         "create service user proms-import-backend \n set ACL on /Questionnaires,/SubjectTypes \n   allow jcr:read for proms-import-backend \n end \n set ACL on /Forms,/Subjects \n   allow jcr:read,rep:write,jcr:versionManagement for proms-import-backend \n end",
         // Trusted users can only access forms submitted by their patients, and delete forms that they created
-        "create group TrustedUsers \n\n set ACL for TrustedUsers \n   deny rep:write on /Subjects \n   allow jcr:removeChildNodes on /Forms \n     deny jcr:removeNode on /Forms restriction(rep:ntNames,cards:Form,cards:Subject) \n     allow jcr:removeNode on /Forms restriction(cards:createdBy) \n    deny jcr:all on /Forms restriction(cards:unsubmittedForms) \n end "
+        "create group TrustedUsers \n\n set ACL for TrustedUsers \n   deny rep:write on /Subjects \n   allow jcr:removeChildNodes on /Forms \n     deny jcr:removeNode on /Forms restriction(rep:ntNames,cards:Form,cards:Subject) \n     allow jcr:removeNode on /Forms restriction(cards:createdBy) \n    deny jcr:all on /Forms restriction(cards:unsubmittedForms) \n end ",
+        // Exporting forms in csv format
+        "create service user csv-export \n set ACL on /Questionnaires,/Forms \n   allow jcr:read for csv-export \n end"
```

The `csv-export` _service user_ will also need `jcr:read` access to `/Subjects` and `/SubjectTypes`.

```suggestion
        "create service user csv-export \n set ACL on /Questionnaires,/Forms,/Subjects,/SubjectTypes \n   allow jcr:read for csv-export \n end"
```